src/plugins/workspace/server/saved_objects/workspace_saved_objects_client_wrapper.ts

```diff
@@ -412,7 +413,7 @@ export class WorkspaceSavedObjectsClientWrapper {
     };
 
     const findWithWorkspacePermissionControl = async <T = unknown>(
-      options: SavedObjectsFindOptions
+      options: SavedObjectsFindOptions & Pick<WorkspaceFindOptions, 'permissionModes'>
```

Comment:
Why we need to add `permissionModes` to options? I think `permissionModes` is already exists in `options.ACLSearchParams`.

Reply:
I will remove the useless code.